schemas/tls.yml

```diff
@@ -0,0 +1,174 @@
+---
+- name: tls
+  title: TLS
+  group: 2
+  short: Fields describing a TLS connection.
+  description: >
+    Fields related to TLS activity.
+  type: group
+  fields:
+
+    - name: version.protocol
+      level: extended
+      type: keyword
+      description: Normalized protocol name parsed from original string (e.g. ssl, tls).
+
+    - name: version.number
+      level: extended
+      type: keyword
+      description: Numeric part of the version parsed from the original string (e.g. 1.2, 3).
+
+    - name: cipher
+      type: keyword
+      level: extended
+      description: String indicating the cipher used during the current connection.
+
+    - name: curve
+      type: keyword
+      level: extended
+      description: String indicating the curve used for the given cipher, when applicable.
+
+    - name: resumed
+      type: boolean
+      level: extended
+      description: Boolean flag indicating if this TLS connection was resumed from an existing TLS negotiation.
+
+    - name: next_protocol
```

Comment:
This field should also mention that the value must be lowercased, just like the slew of other protocol fields I mentioned in the comment above :-)